sale_crm_medical_prescription/tests/test_crm_lead.py

```diff
@@ -24,21 +24,24 @@ def setUp(self):
 
     def test_compute_prescription_order(self):
         """ Test prescription orders properly calculated """
-        self.assertEquals(
-            [self.rx_order_9.id, self.rx_order_10.id],
-            self.crm_lead_1.prescription_order_ids.ids
-        )
+        rx_orders = [self.rx_order_9.id, self.rx_order_10.id]
+        for rx in rx_orders:
```

Comment:
Nice yeah this is a much better solution that what I've been doing. Directly comparing the arrays has become problematic.